src/main/java/org/whispersystems/textsecuregcm/auth/AuthorizationToken.java

```diff
@@ -2,28 +2,27 @@
 
 
 import com.fasterxml.jackson.annotation.JsonProperty;
-import org.apache.commons.codec.DecoderException;
-import org.apache.commons.codec.binary.Hex;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.whispersystems.textsecuregcm.util.Util;
-
-import javax.crypto.Mac;
-import javax.crypto.spec.SecretKeySpec;
-import java.security.InvalidKeyException;
-import java.security.MessageDigest;
-import java.security.NoSuchAlgorithmException;
-import java.util.concurrent.TimeUnit;
 
 public class AuthorizationToken {
```

Comment:
Let's rename this DirectoryCredentials since it's no longer a token

Reply:
done in 0973896e934a0a5e4355b99e041362f437ecb162